sklearn/base.py

```diff
@@ -509,12 +509,12 @@ def get_submatrix(self, i, data):
         ----------
         i : int
             The index of the cluster.
-        data : array
+        data : array-like
             The data.
 
         Returns
         -------
-        submatrix : array
+        submatrix : array-like
```

Comment:
In this case, the type can be more specific:

```suggestion
        submatrix : ndarray
```

Reply:
Good catch! Corrected.